### DISCUSSION TITLE
Win10 taskbar on Win11 - Unable to change which tray icons are shown or hidden

### DISCUSSION THREAD

**Discussion by rgrishanov** [valinet/ExplorerPatcher#2851] [General]
![image](https://github.com/valinet/ExplorerPatcher/assets/23342940/c1edb9ad-0c17-4090-846a-b39af81f0d78)
all the icons ended up hidden, and the menu to change which icons should show, leads to the base Win11 config, which is not respected by the Win10 bar. 

Am I missing something, or this is it for now?
Because normal Win11 config does not influence what actually happens on the Win10 bar:
![image](https://github.com/valinet/ExplorerPatcher/assets/23342940/7d5b931e-9c9a-4ff1-ae76-2b5b4a7550fb)

**Comment by milestprower92** [comment]
that's because windows 11 22H2 and higher no longer uses the windows 10 tray icons on the new taskbar and that's why it doesn't work with the old taskbar anymore

**Reply by rgrishanov** [reply]
So currently there is no other way to tell the win10 taskbar, which icons I want to see permanently, and which to hide? 
Is it impossible atm, or will it be fixed eventually?

**Reply by milestprower92** [reply]
Maybe it can be possible but you need to swap files with windows 10 ones but that's not the purpose of explorer patcher and also that you can break your installation by doing file swapping

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1454

Please close this one.

**Reply by rgrishanov** [reply]
Well, this here is not an issue, but a discussion. But anyway, all I wanted and needed was where you showed, ty, closing this :)
